src/js/edu-benefits/1995/containers/ConfirmationPage.jsx

```diff
@@ -71,26 +82,27 @@ class ConfirmationPage extends React.Component {
             </li>
           </ul>
         </div>
-        <div className="inset secondary expandable">
-          <ExpandingGroup open={this.state.isExpanded} showPlus>
-            <div onClick={this.handleClick} className="clickable">
-              <b>No documents required at this time</b>
-            </div>
-            <div>
-              <p>In the future, you might need:</p>
-              <ul>
-                <li>Your reserve kicker</li>
-                <li>Documentation of additional contributions that would increase your monthly benefits.</li>
-              </ul>
-              <p>Documents can be uploaded using the <a href="https://gibill.custhelp.com/app/utils/login_form/redirect/account%252">GI Bill site</a>.</p>
-            </div>
-          </ExpandingGroup>
+        <div id="collapsiblePanel" className="usa-accordion-bordered form-review-panel">
```

Comment:
This section is copied from `src/js/common/schemaform/review/ReviewCollapsibleChapter.jsx`. That particular component isn't able to be used from the confirmation page, but I can pull this chunk out into a common component if desired.